content/components/accordion/rationale/usage.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: component/rationale
+heading: When to use accordions
+---
+
+We recommend that accordions be used sparingly for primary content on a page.  While they can be appropriate for organising small, specific sections of content, accordions are not a suitable replacement for well formattted plain text. If a user requires all, or most of the information on the page it should be visible, not hidden inside an accordion. 
+
+[Things to consider](https://www.nngroup.com/articles/accordions-complex-content/) when using accordions are, 
+
+**Hiding content makes it harder for a user to scan a webpage.**  If content is hidden inside an accordion it can be difficult for a user to scan a whole web page for content relevent to them. It also makes it hard for a user to use `Control + F` to search for certain keywords. 
```

Comment:
relevent >> relevant

"Web browser's 'Find on page...' search functions do not detect content hidden by accordions, making it harder for users to use their existing tools and behaviours to locate text."